test/unit/event.js

```diff
@@ -2955,54 +2945,16 @@ QUnit.test( "VML with special event handlers (trac-7071)", function( assert ) {
 	ns.remove();
 } );
 
-QUnit.test( "Check order of focusin/focusout events", function( assert ) {
```

Comment:
Has the order changed? Why is this being removed?

Reply:
I guess it's fine as we never guaranteed the order and it was different in IE anyway but I'd like to make it clear we're changing it.

@jquery/core what do you think?

Reply:
If the order changes, this has the potential to break existing code. That kinda smells of a major version to me.

Reply:
I don't think we've ever documented a guaranteed order, and moreover I don't think we're consistent across supported browsers right now. But I can try restoring this test to see what happens.

Reply:
Technically speaking it's not a breaking change as we haven't documented the order. That said, if something is consistent between browsers, people tend to start depending on that and changing the behavior may break them. @gibson042 is right the order is different between browsers right now but it's only different between IE & the rest. If a site doesn't support IE (there are more & more of such sites), for their purposes the current order is consistent between browsers. So there's risk involved here.

Reply:
To add to the above, while the current order is inconsistent between IE & others, what's consistent (& verified by the removed test) is that `focusin` fires before `focus` & `focusout` before `blur`.